wire-ios/Wire-iOS/Sources/URLActionRouter.swift

```diff
@@ -133,6 +133,46 @@ class URLActionRouter: URLActionRouterProtocol {
 // MARK: - PresentationDelegate
 extension URLActionRouter: PresentationDelegate {
 
+    func showPasswordPrompt(for conversationName: String, completion: @escaping (String?) -> Void) {
+        typealias ConversationAlert = L10n.Localizable.Join.Group.Conversation.Alert
+
+        let alertController = UIAlertController(
+            title: ConversationAlert.title(conversationName),
+            message: ConversationAlert.message,
+            preferredStyle: .alert
+        )
+
+        alertController.addTextField { textField in
+            textField.placeholder = ConversationAlert.Textfield.placeholder
+            textField.isSecureTextEntry = true
+        }
+
+        let joinAction = UIAlertAction(title: ConversationAlert.JoinAction.title, style: .default) { _ in
+            let password = alertController.textFields?.first?.text
+            completion(password)
+        }
+
+        let helpLinkURL = URL(string: "https://support.wire.com/hc/en-us/articles/360000574069-Create-guest-rooms")!
```

Comment:
You should add this link to the` extension URL `. We have many links with` https://support.wire.com/...`

Reply:
Done! https://github.com/wireapp/wire-ios/commit/b2e07456455f9f4add2e1b10cefa94e6f0ccd250